examples/ExtendedTutorials/Breakout.h

```diff
@@ -0,0 +1,135 @@
+/*
+Bullet Continuous Collision Detection and Physics Library
+Copyright (c) 2015 Google Inc. http://bulletphysics.org
+
+This software is provided 'as-is', without any express or implied warranty.
+In no event will the authors be held liable for any damages arising from the use of this software.
+Permission is granted to anyone to use this software for any purpose, 
+including commercial applications, and to alter it and redistribute it freely, 
+subject to the following restrictions:
+
+1. The origin of this software must not be misrepresented; you must not claim that you wrote the original software. If you use this software in a product, an acknowledgment in the product documentation would be appreciated but is not required.
+2. Altered source versions must be plainly marked as such, and must not be misrepresented as being the original software.
+3. This notice may not be removed or altered from any source distribution.
+*/
+
+#ifndef ET_BREAKOUT_EXAMPLE_H
+#define ET_BREAKOUT_EXAMPLE_H
+
+#include "btBulletDynamicsCommon.h"
+
+struct vec2 // your engine-independent object representation 2D vector struct
+{
+	vec2(float x, float y)
+	{
+		p[0] = x;
+		p[1] = y;
+	}
+	float p[2];
+};
+
+struct vec3 // your engine-independent object representation 3D vector struct
+{
+	vec3(float x,float y, float z)
+	{
+		p[0] = x;
+		p[1] = y;
+		p[2] = z;
+
+	}
+
+	float p[3];
+};
+
+struct Vertex // you engine-independent object representation vertex struct
+{
+	Vertex(vec3 p):position(p), uv(0,0), color(0,0,0) {
+	}
+
+    vec3 position; // vertex position
+    vec2 uv; // bullet physics browser does not take any textures
+	vec3 color; // bullet physics browser does not take any colors
+};
+
+/**
+ * Game Object base class
+ *
+ * All game objects inherit from this class.
+ */
+class GameObject {
+public:
+	GameObject():m_position(0,0,0),m_orientation(0,0,0,1),m_body(0),m_shape(0),m_tag(PADDLE) {}
+	virtual ~GameObject() {}
+	void createShapeWithVertices(const Vertex* vertices, unsigned int vertexCount,bool convex);
+	void createBodyWithMass(float mass);
+
+	void setPosition(btVector3 position);
```

Comment:
We need to pass btVector3 and btQuaternion using const reference, otherwise SSE/SIMD compilers break the build.
Here is the error message: >..\..\examples\ExtendedTutorials\Breakout.cpp(366): error C2719: 'position': formal parameter with requested alignment of 16 won't be aligned

 Can you please change it into: void setPosition(const btVector3& position); and void setOrientation(const btQuaternion& orientation);